webapp/pom.xml

```diff
@@ -38,6 +121,26 @@
     </build>
 
     <dependencies>
+        <dependency>
+            <groupId>org.webjars.npm</groupId>
+            <artifactId>jquery-migrate</artifactId>
+            <version>3.4.1</version>
+        </dependency>
+        <dependency>
+            <groupId>org.mvnpm</groupId>
```

Comment:
@litvinovg did you investigate difference between this artifact in org.mvnpm and org.webjars (https://mvnrepository.com/artifact/org.webjars.npm/datatables.net/2.0.5)?

Reply:
As you can see on this page version 2.0.5 is still not available for org.webjars
https://mvnrepository.com/artifact/org.webjars/datatables

Reply:
let's keep group id which is set at the moment